### PR TITLE
- jsk_tools/src/test_topic_published.py: set default timeout to 10 sec

### PR DESCRIPTION
**Pull request by k-okada**
- jsk_tools/src/test_topic_published.py: set default timeout to 10 sec
- jsk_tools/src/sanity_lib.py: add timouetout informatoin

### CODE REVIEW

**Comment by k-okada** [comment]
@wkentaro , please review

File: jsk_tools/src/jsk_tools/sanity_lib.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def __init__(self, topic_name, timeout=5, echo=False, data_class=None):
         self.launched_time = rospy.Time.now()
         self.first_time_callback = True
         self.echo = echo
-        print " Checking %s" % (topic_name)
+        print " Checking %s at %d Hz" % (topic_name, timeout)
```
Comment (wkentaro) [inline jsk_tools/src/jsk_tools/sanity_lib.py]: " Checking %s for %d seconds' ?

Comment (k-okada) [inline jsk_tools/src/jsk_tools/sanity_lib.py]: correct

Comment (wkentaro) [inline jsk_tools/src/jsk_tools/sanity_lib.py]: at -> for?

**Comment by wkentaro** [comment]
LGTM

**Pull request merged**